tasks/workers/etl/sandbox/etl.js

```diff
@@ -64,14 +64,54 @@ function getUrlFromNodeId(node){
   });
 }
 
+function getDateId(date,interval){
+  return interval == "day" ? date.toLocaleDateString():date.toLocaleDateString()+ ":" + date.getHours();
+}
+
+function buildTimeseries(nodes){
+  var dateMap = {};
+  var firstDate;
+  for(var i =0; i< nodes.length; i++){
+    var node = nodes[i];
+    var date = new Date(node.time * 1000);
+    if(!firstDate || node.time < firstDate){
+      firstDate = node.time;
+    }
+    var dateToHour =  new Date(date.getFullYear(), date.getMonth(), date.getDate(), date.getHours());
```

Comment:
yeah I didn't know moment so it was a get to demoable state first issue...there are some other things in there I want to fix as well...like multiple ways to display data and how data is passed around...messages vs scope...
It's not done by any means.